Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Based off https://medium.com/@chemidy/create-the-smallest-and-secured-golang-docker-image-based-on-scratch-4752223b7324
+
+# STEP 1 build executable binary
+FROM golang:alpine as builder
```

Comment:
With alpine version like: `1.11.2-alpine` it will be possible to control what Go version will be use for a build.